wordle-web/src/views/AboutView.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="about">
+    <v-card>
+      <v-card-title> This is a vue/vuetify class! </v-card-title>
+      <v-card-subtitle> (College Class) </v-card-subtitle>
+
+      <v-card-text>
+        Lorem ipsum dolor sit amet blah blah blah
+        <v-icon icon="mdi:mdi-abacus" />
+      </v-card-text>
+
+      <v-card-actions>
+        <v-spacer />
+        <v-btn text color="warning">Cancel</v-btn>
+        <v-btn text color="primary">OK</v-btn>
```

Comment:
Buttons dont have any associated actions, I would just replace these two with a single button that uses the router to navigate back to the Home Page view

Reply:
I'll leave them in in case they come in handy later.